### PR TITLE
Document GitHub apps/integrations

### PR DESCRIPTION
**Pull request by austinlparker**
Related to #1844

This is incomplete as of now, we need help to fill it out and to perform appropriate security reviews.

### CODE REVIEW

File: tools/github/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# GitHub Applications and Integrations
+
+This document and directory contain information about all configured GitHub
+applications and integrations with the OpenTelemetry organization.
+
+| Name             | Description                                          | Scopes                      | Notes                                                                  |
+|------------------|------------------------------------------------------|-----------------------------|------------------------------------------------------------------------|
+| Azure Pipelines  | Support for Azure Pipelines                          | Org-wide                    | Included by default                                                    |
```
Comment (svrnm) [inline tools/github/README.md:8]:
```suggestion
| [Azure Pipelines](https://github.com/apps/azure-pipelines)  | Support for Azure Pipelines                          | Org-wide                    | Included by default                                                    |
```

File: tools/github/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| ciMate           | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:9]:
```suggestion
| [ciMate](https://github.com/apps/cimate)           | TBD                                                  | TBD                         | TBD                                                                    |
```

This one seems to be dead?

File: tools/github/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+| CircleCI Checks  | Integration with CircleCI                            | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:10]:
```suggestion
| [CircleCI Checks](https://github.com/apps/circleci-checks)  | Integration with CircleCI                            | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+| Codecov          | Integration with Codecov for PR code coverage checks | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:11]:
```suggestion
| [Codecov](https://github.com/apps/codecov)          | Integration with Codecov for PR code coverage checks | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+| Forking Renovate | Dependecy updates via Renovate                       | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:12]:
```suggestion
| [Forking Renovate](https://github.com/apps/forking-renovate) | Dependency updates via Renovate                       | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+| EasyCLA          | Required CLA checking from LF                        | Org-wide                    | This must be enabled for all repositories for PRs against main branch. |
```
Comment (svrnm) [inline tools/github/README.md:13]:
```suggestion
| [EasyCLA](https://github.com/apps/linux-foundation-easycla)          | Required CLA checking from LF                        | Org-wide                    | This must be enabled for all repositories for PRs against main branch. |
```

File: tools/github/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+| Mergify          | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:14]:
```suggestion
| [Mergify](https://github.com/apps/mergify)          | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+| Netlify          | Integration with Netlify for website deployments     | opentelemetry.io repository | n/a                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:15]:
```suggestion
| [Netlify](https://github.com/apps/netlify)          | Integration with Netlify for website deployments     | opentelemetry.io repository | n/a                                                                    |
```

File: tools/github/README.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+| Octobox          | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:16]:
```suggestion
| [Octobox](https://github.com/apps/octobox)          | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+| OpenShift CI     | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:17]:
```suggestion
| [OpenShift CI ](https://github.com/apps/openshift-ci)    | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+| Polls            | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:18]:
```suggestion
| [Polls](https://github.com/apps/polls)            | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+| project-bot      | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:19]:
```suggestion
| [project-bot](https://github.com/apps/project-bot)      | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+| Renovate         | Dependecy updates via Renovate                       | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:20]:
```suggestion
| [Renovate](https://github.com/apps/renovate)         | Dependency updates via Renovate                       | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+| Scope App        | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:21]:
```suggestion
| [Scope App](https://github.com/apps/scope-app)        | TBD                                                  | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+| Slack            | Slack integration for GitHub                         | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:22]:
```suggestion
| [Slack](https://github.com/apps/slack)            | Slack integration for GitHub                         | TBD                         | TBD                                                                    |
```

File: tools/github/README.md [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+| Stale            | Stale issue bot                                      | Org-wide                    | n/a                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:23]:
```suggestion
| [Stale](https://github.com/apps/stale)            | Stale issue bot                                      | Org-wide                    | n/a                                                                    |
```

File: tools/github/README.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+| Welcome          | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (svrnm) [inline tools/github/README.md:24]:
```suggestion
| [Welcome](https://github.com/apps/welcome)          | TBD                                                  | TBD                         | TBD                                                                    |
```

**Review by svrnm — CHANGES_REQUESTED** [review]
My review comments add the links to the (hopefully) correct github apps, to make it easier to identify them. While doing so, it looks like ciMate is dead and stale bot states that it is deprecated and should be replaced by the stale gh action.

**Review by trask — COMMENTED** [review]
@open-telemetry/technical-committee can you verify the list and let us know the scopes that are currently set for each?

**Comment by reyang** [comment]
> @open-telemetry/technical-committee can you verify the list and let us know the scopes that are currently set for each?

| Name | Description | Scopes | Notes |
| - | - | - | - |
| Azure Pipelines  | Support for Azure Pipelines | open-telemetry/opentelemetry-dotnet, open-telemetry/opentelemetry-dotnet-contrib, open-telemetry/opentelemetry-dotnet-instrumentation | Included by default |
| ciMate | TBD | open-telemetry/opentelemetry-java-instrumentation | TBD |
| CircleCI Checks | Integration with CircleCI | all repositories | TBD |
| Codecov | Integration with Codecov for PR code coverage checks | all repositories | TBD |
| Forking Renovate | Dependency updates via Renovate | open-telemetry/opentelemetry-js, open-telemetry/opentelemetry-js-contrib | TBD |
| EasyCLA | Required CLA checking from LF | all repositories | This must be enabled for all repositories for PRs against main branch. |
| Mergify | TBD | open-telemetry/opentelemetry-operator | TBD |
| Netlify | Integration with Netlify for website deployments | all repositories | n/a |
| Octobox | TBD | all repositories | TBD |
| OpenShift CI | TBD | open-telemetry/opentelemetry-operator | TBD |
| Polls | TBD | all repositories | TBD |
| project-bot | TBD | open-telemetry/opentelemetry-collector, open-telemetry/opentelemetry-collector-contrib | TBD |
| Renovate | Dependency updates via Renovate | open-telemetry/opentelemetry-java, open-telemetry/opentelemetry-erlang, open-telemetry/opentelemetry-collector, open-telemetry/opentelemetry-collector-contrib, open-telemetry/opentelemetry-java-instrumentation, open-telemetry/opentelemetry-java-contrib, open-telemetry/opentelemetry-erlang-contrib, open-telemetry/opentelemetry-java-examples, open-telemetry/opentelemetry-demo, open-telemetry/opentelemetry-android | TBD |
| Scope App | TBD | open-telemetry/opentelemetry-swift | TBD |
| Slack | Slack integration for GitHub | all repositories | TBD |
| Stale | Stale issue bot | all repositories | n/a |
| Welcome | TBD | open-telemetry/opentelemetry-php, open-telemetry/opentelemetry-php-contrib | TBD |

**Comment by trask** [comment]
> | Name | Description | Scopes | Notes |
> | - | - | - | - |
> ciMate | TBD | open-telemetry/opentelemetry-java-instrumentation | TBD |

sent #1849 to remove this as it's no longer being used

**Comment by svrnm** [comment]
What are our next steps on this? Anything I can help with? Or is this something that can be handed over to SIG security?

**Comment by reyang** [comment]
FYI - most likely we don't need Azure Pipelines anymore. I'm confirming with the maintainers to see if we can remove it.

image

**Comment by reyang** [comment]
Confirmed with repo maintainers and removed the Azure Pipeline since it's not needed.

**Review by trask — APPROVED** [review]

File: tools/github/README.md [hunk 18]
```diff
@@ -0,0 +1,24 @@
+# GitHub Applications and Integrations
+
+This document and directory contain information about all configured GitHub
+applications and integrations with the OpenTelemetry organization.
+
+| Name             | Description                                          | Scopes                      | Notes                                                                  |
+|------------------|------------------------------------------------------|-----------------------------|------------------------------------------------------------------------|
+| [Azure Pipelines](https://github.com/apps/azure-pipelines)  | Support for Azure Pipelines                          | Org-wide                    | Included by default                                                    |
```
Comment (trask) [inline tools/github/README.md:8]: https://github.com/open-telemetry/community/pull/1845#issuecomment-1909098871
```suggestion
```

File: tools/github/README.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+| [ciMate](https://github.com/apps/cimate)           | TBD                                                  | TBD                         | TBD                                                                    |
```
Comment (trask) [inline tools/github/README.md:8]: I don't see this(?)

**Review by reyang — APPROVED** [review]

File: tools/github/README.md (hunk 19 above)
Comment (reyang) [reply tools/github/README.md:8]: This is what I'm seeing now:

image

Comment (reyang) [reply tools/github/README.md:8]: Anyways, I don't feel this is a blocker for this PR, as long as we merge the structure, folks can send follow up PRs to keep things up-to-date.

**Comment by austinlparker** [comment]
I've gone thru and updated this as per today, should be good to go.

**Comment by austinlparker** [comment]
@svrnm can you re-review?

**Review by svrnm — APPROVED** [review]

**Pull request merged**